utils/node_local_test.py

```diff
@@ -977,6 +990,43 @@ def run_dfuse(server, conf):
 
     print('Reached the end, no errors')
 
+def double_container_test(server, conf):
+    # TODO: This doesn't work with two pools, there appears to be a bug
```

Comment:
(style)  'doesn'' may be misspelled - perhaps 'doesn't'?
(lint) This doesn't work with two pools, there appears to be a bug